### PR TITLE
fix: do not use functions storage account for storing message content

### PR DESCRIPTION
**Pull request by gunzip**
as for queues, avoid using functions storage account to store message content

### CODE REVIEW

File: lib/created_message_queue_handler.ts [hunk 1]
```diff
@@ -73,7 +73,7 @@ const notificationsCollectionUrl = documentDbUtils.getCollectionUri(
   "notifications"
 );
 
-const storageConnectionString = getRequiredStringEnv("AzureWebJobsStorage");
+const storageConnectionString = getRequiredStringEnv("QueueStorageConnection");
```
Comment (cloudify) [inline lib/created_message_queue_handler.ts]: No documentation to update?

Comment (gunzip) [reply lib/created_message_queue_handler.ts]: there are no references to the different blob storages actually

Comment (cloudify) [reply lib/created_message_queue_handler.ts]: I mean, in the setup instructions

Comment (gunzip) [reply lib/created_message_queue_handler.ts]: same there (no mention of the two storages as its handled by the programmatic tasks)

Comment (gunzip) [reply lib/created_message_queue_handler.ts]: can we merge this one ?

**Pull request merged**